src/edu/csus/ecs/pc2/ui/JCheckBoxJList.java

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and .
```

Comment:
Looks like you delete Troy by accident - please fix.

Reply:
I don't know why it is there. Sorry Troy

Reply:
fixed!